### DISCUSSION TITLE
Wifi on the Orange Pi 5 Pro

### DISCUSSION THREAD

**Discussion by jeremysalwen** [Joshua-Riek/ubuntu-rockchip#955] [Q&A]
Has anyone gotten wifi working on the Orange Pi 5 Pro?  I have tried booting on Ubuntu 24.04 and the wifi doesn't show up in ifconfig.

EDIT: After installing network-manager, I am able to see `nmcli` lists wifi0 and I am able to connect over wifi.

**Comment by AdarWa** [comment]
Thank You!!
